### PR TITLE
Replace pycrypto with pycryptodome

### PR DESCRIPTION
**Pull request by jordanvance**
pycrypto hasn't been updated since 2013. 

https://www.pycryptodome.org/en/latest/

### CODE REVIEW

File: Makefile
Comment (mgedmin) [inline Makefile]: This might still be Python 2 on some distros, better switch to `python3` to be explicit.

Also, the Makefile is purely for maintainer convenience, and I don't think keas.kmi has any active maintainers that like Makefiles, so I suggest removing the Makefile entirely.

File: bootstrap.py
Comment (mgedmin) [inline bootstrap.py]: bootstrap.py is obsolete and doesn't work with modern setuptools versions.  People are advised to pip install zc.buildout somewhere and then run `buildout bootstrap` instead.

We've removed bootstrap.py from many other repositories and I recommend removing it here too.

**Review by mgedmin — CHANGES_REQUESTED** [review]
LGTM, if you also update CHANGES.txt and mention the drop of Python 2 (and 3.4-3.5).

The upcoming version in the changelog should probably be changed to 4.0.0 (and 4.0.0.dev0 in setup.py); the drop of Python 2 compatibility feels major.

I wold also rename README.txt and CHANGES.txt to README.rst and CHANGES.rst, which will require small updates in the long_description= code in setup.py.

The changelog update is the only change that I insist on; I leave all other suggestions for your discretion.

File: setup.py
Comment (mgedmin) [inline setup.py]: (BTW other people might be less than happy with the dropping of Python 2 at this time, see https://github.com/zopefoundation/meta/issues/15 and https://www.zope.org/developer/python2.html.)

File: setup.py
Comment (mgedmin) [inline setup.py:53]: No 3.9?

File: setup.py
Comment (mgedmin) [inline setup.py:50]: It would be good to also remove the unsupported Python versions from .travis.yml.

It would be even better to remove .travis.yml (which doesn't work anyway because the free credits ran out) and set up GitHub Actions.  We have templates and scripts in https://github.com/zopefoundation/meta/tree/master/config for automating that.

File: bootstrap.py
Comment (jordanvance) [reply bootstrap.py]: it will be interesting to learn this. will put it on my docket for some time soon.

File: setup.py
Comment (jordanvance) [reply setup.py]: decided not to drop it.

File: setup.py
Comment (jordanvance) [reply setup.py:50]: ditto again for the when i have spare cycles.

File: setup.py
Comment (jordanvance) [reply setup.py:53]: added

**Comment by icemac** [comment]
@jordanvance Thank you for your contribution.

According to the contributing policies of the zopefoundation organization you need to sign a contributor agreement before any non-trivial change before it can be merged. For details please consult the [Contributing guidelines for zopefoundation projects](https://github.com/zopefoundation/.github/blob/master/CONTRIBUTING.md).

File: bootstrap.py
Comment (strichter) [reply bootstrap.py]: I think we should just drop buildout and only have a tox setup. People can then use `venv` and `buildout if they want to.

**Review by mgedmin — APPROVED** [review]

**Comment by jordanvance** [comment]
> @jordanvance Thank you for your contribution.
> 
> According to the contributing policies of the zopefoundation organization you need to sign a contributor agreement before any non-trivial change before it can be merged. For details please consult the [Contributing guidelines for zopefoundation projects](https://github.com/zopefoundation/.github/blob/master/CONTRIBUTING.md).

Done.

**Comment by jugmac00** [comment]
@jordanvance Once you get notice that the contribution agreement got accepted you can and should merge your pull request yourself.

**Pull request merged**
